### ISSUE TITLE
Why policy loss is negative?

### ISSUE THREAD

**Issue by xuehy** [ikostrikov/pytorch-a3c#30]
In train.py, the code for updating the policy loss is
```
 policy_loss = policy_loss - \
                log_probs[i] * Variable(gae) - 0.01 * entropies[i]
```

According to the original paper, I think it should be 
```
 policy_loss = policy_loss + \
                log_probs[i] * Variable(gae) + 0.01 * entropies[i]
```

**Comment by Kaixhin** [comment]
In the paper, the equations are written assuming gradient **ascent**. As we use gradient **descent** optimisers, the equations have to be negated.
